src/main/webapp/style/base/_typography.css

```diff
@@ -49,6 +49,13 @@ body {
   width: var(--panel-icon-width);
 }
 
+.panel__optimize-button {
```

Comment:
So this is coming to master soon in PR #125, but there are some existing buttons you can put at the bottom of this panel. 

Alternatively, you can use this button no problem, but I would try to fix up the alignment. It seems to not be vertically centered in its container

Reply:
Also, it may be better to move this button to the bottom of the panel. If a tasklist name is long, it might push your button off the panel or otherwise squash it. 

You could even use the "Push to Start" button that's currently in the "Assign" panel and just create a content entry for this.

Reply:
I think this is slightly better now(?)

![Screenshot 2020-08-06 at 2 18 36 AM](https://user-images.githubusercontent.com/46367855/89497921-45d16c00-d78b-11ea-9b28-d0c986a45f92.png)

Reply:
Yep it is! Small nit picks: 
1) Make the button green (there's an existing class for this, `panel__green-button`)
2) If you want, if you found a way to overwrite the `width` property of the button to be `100%` (you could make another class for this if you wanted by adopting most of the `button-1-of-2` properties and changing the width. The new class could be `button-1-of-1`)

Reply:
+1 to making the optimize button green

Reply:
Thanks for the suggestions! I made a new class and put 30% instead - not a designer by any means but I just thought 100% made it look a little weird.

![Screenshot 2020-08-06 at 5 38 52 PM](https://user-images.githubusercontent.com/46367855/89585528-ff702180-d80b-11ea-8759-a5e469be694b.png)